awscli/customizations/s3/utils.py

```diff
@@ -353,6 +353,30 @@ def relative_path(filename, start=os.path.curdir):
         return os.path.abspath(filename)
 
 
+def set_file_utime(filename, desired_time):
+    """
+    Set the utime of a file, and if it fails, raise a more explicit error.
+
+    :param filename: the file to modify
+    :param desired_time: the epoch timestamp to set for atime and mtime.
+    :raises: SetFileUtimeError: if you do not have permission (errno 1)
+    :raises: OSError: for all errors other than errno 1
+    """
+    try:
+        os.utime(filename, (desired_time, desired_time))
+    except OSError as e:
+        # Only raise a more explicit exception when errno is 1
+        if e.errno is not 1:
```

Comment:
Very small thing (it does not need to change). Just want to give a heads up. I usually avoid using ``is`` unless I am actually checking that the identity of an object is the identity of another object. It is fine for small integers like these, but for larger ones there may be edge cases:
```py
In [1]: 1000 is 10**3
Out[1]: False
```
Equality checks are usually safer if you are just checking values.